client/lib/explat/internals/log-error.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { getLogger } from 'calypso/server/lib/logger';
+
+// Using typescript to ensure we are being safe in SSR
+declare const window: undefined | ( Window & typeof globalThis );
+
+export const isDevelopmentMode = process.env.NODE_ENV === 'development';
```

Comment:
Does this need to be exported?

Reply:
Yeah, it needs to be somewhere so I stuck it in here.

Reply:
Does it still need to be exported with `logErrorOrThrowInDevelopmentMode()` in this file? We can move the function to this PR so that the `anon-id` PR doesn't touch `log-error`.

Reply:
It is also used in `lib/explat/index.ts`, but I don't mind duplicating it across the files.

Reply:
I am going to fix this in the index PR so I don't have to redo all of these

Reply:
See https://github.com/Automattic/wp-calypso/pull/50474